plugins/flutter_aepcore/README.md

```diff
@@ -20,51 +20,150 @@ Install instructions for this package can be found [here](https://pub.dev/packag
 
 > Note: After you have installed the SDK, don't forget to run `pod install` in your `ios` directory to link the libraries to your Xcode project.
 
-## Usage
+## Importing the Plugins
 
-### Initializing
+Import the package in your **Dart** code as follows:
 
-Initializing the SDK should be done in native code (AppDelegate / SceneDelegate for iOS and Application class for Android). Documentation for initializing the SDK can be found [here](https://developer.adobe.com/client-sdks/documentation/getting-started/get-the-sdk/#2-add-initialization-code).
+```dart
+import 'package:flutter_{extension}/flutter_{plugin_name}.dart'
+```
 
-As part of the initialization code, make sure that you set the SDK wrapper type to `Flutter` before you start the SDK.
+## Initializing
 
-Refer to the [Initialization](https://github.com/adobe/aepsdk_flutter#initializing) section of the root README for more information about initializing the SDK.
+To initialize the SDK, use the following methods:
+- [MobileCore.initializeWithAppId(appId)](#initializewithappid)
+- [MobileCore.initialize(initOptions)](#initialize)
 
-### Core
+Refer to the root [Readme](https://github.com/adobe/aepsdk_flutter/blob/main/README.md) for more information about the SDK setup.
+
+## Core
 
 For more detailed information on the Core APIs, visit the documentation [here](https://developer.adobe.com/client-sdks/documentation/mobile-core/)
 
-##### Importing Core:
+## Importing Core:
+In your Flutter application, import the Core package as follows:
+
 ```dart
 import 'package:flutter_aepcore/flutter_aepcore.dart';
 ```
 
-##### Getting Core version:
- ```dart
+## API reference
+
+### extensionVersion
+Returns the SDK version of the Core extension.
+
+**Syntax**
+```dart
+Future<String> get extensionVersion
+```
+
+**Example**
+```dart
 String version = await MobileCore.extensionVersion;
- ```
+```
+### initializeWithAppId
+It initializes AEP SDKs by automatically registering all extensions bundled with the application and enabling automatic lifecycle tracking by default.
```

Comment:
Initialize the AEP SDK by automatically registering all extensions bundled with the application and enabling automatic lifecycle tracking.